### ISSUE TITLE
Change amount at cart scroll bar?

### ISSUE THREAD

**Issue by river7527** [sivadass/react-shopping-cart#7]
Can you advise what to change and make the amount changeable within the cart scrollbar?

**Comment by sivadass** [comment]
You want the product quantity to be updated inside cart scrollbar?

**Comment by river7527** [comment]
That's exactly. Just wondering sometime people may accidentally add a larger amount than expected, while then they have to remove the products and add the ones again from scratch.
I think it would be more convenient if the amounts insider cart scrollbar are changeable.

**Comment by sivadass** [comment]
Yeah it will really improve the experience. But it will need a major code change, as I am currently updating the product quantity only when add to cart is pressed, but now it has to be updated when both stepper inputs (Increase and decrease) are pressed. I will try to add this feature as soon as possible. In the mean time, you just try and give a pull request, am happy to help you further. I am trying in [this](https://github.com/sivadass/react-shopping-cart/tree/NewFeatureRequest) branch.
